specification/app/resource-manager/Microsoft.App/preview/2022-01-01-preview/examples/ContainerApps_ListSecrets.json

```diff
@@ -0,0 +1,23 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "rg",
+    "name": "testcontainerApp0",
+    "api-version": "2021-03-01"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "value": [
+          {
+            "name": "secret1"
```

Comment:
>  "name": "secret1" [](http://example.com/codeflow?start=10&length=19)

Should this include "value" too?

Reply:
we should not include value here as it is secret. otherwise CI will fail.